### ISSUE TITLE
Validation fails without alias

### ISSUE THREAD

**Issue by SebTM** [symfony/flex#349]
Hello,

for testing I was only able to create a public repository a recipe - because I don't have a paid account on GitHub I was unable to create the repository as private. (Again: It is only for testing purpose!)

As you can see here: https://github.com/SebTM/symfony-recipes/pull/2
I get the following error: 
"Recipe is not needed as it only registers bundle "SebTM\INWX\InwxApiBundle" for all environments"

My Recipe as what you can see, also manages some environment-variables - so the recipe is needed? Or can you explain me how this would be done else?

Best Regards

**Comment by weaverryan** [comment]
Hmm, seems legit - the recipe does indeed include some env var instructions, or perhaps I’m also missing something :)

**Comment by SebTM** [comment]
Thx for having a look at it @weaverryan 👍 
How can we go on with this issue? It's a bit a blocker for me as on "symfony/recipes-contrib" I'm not allowed to use aliases as I found out in the documentation. The only was would be to copy a config-directory with dummy-file but this seems to be no useful solution.

Is the code of the bot open-source? As I found nothing about the error here in flex nor a repository of the bot/recipe-server.
